trillian/integration/ct_integration_test.cfg

```diff
@@ -1,26 +1,39 @@
-[
-	{
-		"LogID": @TREE_ID@,
-		"Prefix": "athos",
-		"RootsPEMFile": ["@TESTDATA@/fake-ca.cert"],
-		"PubKeyPEMFile": "@TESTDATA@/ct-http-server.pubkey.pem",
-		"PrivKeyPEMFile": "@TESTDATA@/ct-http-server.privkey.pem",
-		"PrivKeyPassword": "dirk"
+{
+	"log_id": @TREE_ID@,
+	"prefix": "athos",
+	"roots_pem_file": ["@TESTDATA@/fake-ca.cert"],
+	"public_key": {
+		"der": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEB/hRr6qMVoOQMbeA49Ya9y82BnHs3Tu+fjZvDRwcYAt/9Z//5SRJNFbySxBfvwgf+Q7PNbWKioswClS3vx1NuQ=="
 	},
-	{
-		"LogID": @TREE_ID@,
-		"Prefix": "porthos",
-		"RootsPEMFile": ["@TESTDATA@/fake-ca.cert"],
-		"PubKeyPEMFile": "@TESTDATA@/ct-http-server.pubkey.pem",
-		"PrivKeyPEMFile": "@TESTDATA@/ct-http-server.privkey.pem",
-		"PrivKeyPassword": "dirk"
+	"private_key": {
+		"@type": "type.googleapis.com/keyspb.PEMKeyFile",
+		"path": "@TESTDATA@/ct-http-server.privkey.pem",
+		"password": "dirk"
+	}
+}
+{
+	"log_id": @TREE_ID@,
+	"prefix": "porthos",
+	"roots_pem_file": ["@TESTDATA@/fake-ca.cert"],
+	"public_key": {
+		"der": "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEB/hRr6qMVoOQMbeA49Ya9y82BnHs3Tu+fjZvDRwcYAt/9Z//5SRJNFbySxBfvwgf+Q7PNbWKioswClS3vx1NuQ=="
```

Comment:
The mix of explicit DER data and indirect PEM data seems a little jarring (but no need to fix if doing so is inconvenient)